src/constructs/cloudwatch/__snapshots__/lambda-alarms.test.ts.snap

```diff
@@ -199,7 +199,11 @@ Object {
               ],
             },
             Object {
-              "Action": "ssm:GetParametersByPath",
+              "Action": Array [
+                "ssm:GetParametersByPath",
+                "ssm:GetParameters",
+                "ssm:GetParameter",
```

Comment:
I don't strictly need `ssm:GetParameter` for the change I'm working on, but I didn't really see the harm in including it here.